components/FruitBasket.js

```diff
@@ -0,0 +1,32 @@
+const React = require('react');
+const { Component } = require('react');
+
+const Filter = require('./Filter');
+const FilteredFruitList = require('./FilteredFruitList.js');
+
+const FruitBasket = ({
+  fruit,
+  filters,
+  currentFilter,
+  updateFilterCallback
```

Comment:
Can we call this `onUpdateFilter` or `onFilterUpdate` to be more in line with how React defines their handlers?